nixos/modules/system/boot/kexec.nix

```diff
@@ -19,7 +20,7 @@
               exit 0
             fi
 
-            p=$(readlink -f /nix/var/nix/profiles/system)
+            p=$(readlink -f /run/next-system)
```

Comment:
Should this fall back  to `/run/current-system`  if `/run/next-system` doesn't exist?